### DISCUSSION TITLE
turn off the create on *** entry at the beginning of every new page

### DISCUSSION THREAD

**Discussion by joe-pirat** [zim-desktop-wiki/zim-desktop-wiki#2535] [General]
How can this feature be switched off?
I like zim and have used it daily for years.
What bothers me from the start, is that I have to manually delete the "Created ******" on every new page because I like it to be simple and clear and when I created a page is completely irrelevant to me.

**Comment by bruvellu** [comment]
Go to `Edit` > `Templates`, scroll to the bottom and click on `Default` in the `wiki` section, then press `Edit`. The template file will open in your default text editor and you'll see this:

```
======= [% page.basename %] =======
[% gettext("Created") %] [% strftime("%A %d %B %Y") %]
```

Remove the "Created" line to look like this:

```
======= [% page.basename %] =======
```

...then save the file and close the Templates dialog.

The next page you create won't have the "Created" line.

**Comment by joe-pirat** [comment]
thx 
im so happy to get this option .....
